### PR TITLE
Gradle support

### PR DESCRIPTION
**Pull request by Fatme**
https://github.com/NativeScript/nativescript-cli/issues/687

### CODE REVIEW

File: lib/services/android-project-service.ts [hunk 1]
```diff
@@ -436,23 +381,11 @@ class AndroidProjectService extends projectServiceBaseLib.PlatformProjectService
 			return newTarget;
 		}).future<string>()();
 	}
-
-	private updateTarget(projectRoot: string, newTarget: string): IFuture<void> {
+
```
Comment (teobugslayer) [inline lib/services/android-project-service.ts]: Unneeded empty space

**Comment by teobugslayer** [comment]
Can we link to the GitHub issue we are addressing?

**Comment by dtopuzov** [comment]
:+1:

File: lib/services/android-project-service.ts [hunk 2]
```diff
@@ -181,13 +167,29 @@ class AndroidProjectService extends projectServiceBaseLib.PlatformProjectService
 	}
 
 	public buildProject(projectRoot: string): IFuture<void> {
-		let buildConfiguration = this.$options.release ? "release" : "debug";
-		let args = this.getAntArgs(buildConfiguration, projectRoot);
-		return this.spawn('ant', args);
+		return (() => {
+			this.$projectDataService.initialize(this.$projectData.projectDir);
+			let frameworkVersion = this.$projectDataService.getValue(this.platformData.frameworkPackageName).wait().version;
+			if(semver.lt(frameworkVersion, "1.3.0")) {
```
Comment (teobugslayer) [inline lib/services/android-project-service.ts]: 1.3.0 is a magic constant - consider naming it

**Comment by teobugslayer** [comment]
Linter messages:
```
>> lib/services/android-project-service.ts[76, 1]: trailing whitespace
>> lib/services/android-project-service.ts[79, 12]: trailing whitespace
>> lib/services/android-project-service.ts[262, 1]: trailing whitespace
>> lib/services/android-project-service.ts[415, 1]: trailing whitespace
>> lib/services/doctor-service.ts[73, 3]: if statements must be braced
>> lib/services/doctor-service.ts[74, 1]: trailing whitespace
```

**Comment by Fatme** [comment]
I fixed all lint messages. Just get the latest changes.

**Comment by teobugslayer** [comment]
:+1:

**Pull request merged**